practicals/GNN_for_recommendations.ipynb

```diff
@@ -0,0 +1,1284 @@
+{
```

Comment:
<p>Do you perhaps have an example KG particular to a movie dataset? This may be a bit confusing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>I couldnt get a better image, do you have a better recommendations and send me through?</p>

Reply:
<p>may be i can just remove throughout.. will do that in the next commit for now :) </p>